app/components/at_mention/at_mention.js

```diff
@@ -48,28 +51,18 @@ export default class AtMention extends React.PureComponent {
     }
 
     goToUserProfile = () => {
-        const {navigator, theme} = this.props;
+        const {actions} = this.props;
         const {intl} = this.context;
-        const options = {
-            screen: 'UserProfile',
-            title: intl.formatMessage({id: 'mobile.routes.user_profile', defaultMessage: 'Profile'}),
-            animated: true,
-            backButtonTitle: '',
-            passProps: {
-                userId: this.state.user.id,
-            },
-            navigatorStyle: {
-                navBarTextColor: theme.sidebarHeaderTextColor,
-                navBarBackgroundColor: theme.sidebarHeaderBg,
-                navBarButtonColor: theme.sidebarHeaderTextColor,
-                screenBackgroundColor: theme.centerChannelBg,
-            },
+        const screen = 'UserProfile';
+        const title = intl.formatMessage({id: 'mobile.routes.user_profile', defaultMessage: 'Profile'});
+        const passProps = {
+            userId: this.state.user.id,
         };
 
         if (Platform.OS === 'ios') {
-            navigator.push(options);
+            actions.goToScreen(screen, title, passProps);
         } else {
-            navigator.showModal(options);
+            actions.showModal(screen, title, passProps);
```

Comment:
is this still needed on Android? this was on RNNv1 cause of some weird issues with the previous version, can we confirm this is still needed, if not just remove